runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZonePlugin.java

```diff
@@ -192,6 +212,26 @@ else if (msg.contains("Ultimate force"))
 				}
 			}
 		}
+		else if (msg.contains("You drink some of your overload potion."))
+		{
+			lastOverload = Instant.now();  // Save time of last overload
+			overloadNotificationSend = true;  // Queue up a overload notification once time threshold is reached
+		}
+
+	}
+
+	private void checkOverload()
+	{
+		// overloadNotificationSend is already true since this method is being called
```

Comment:
Probably the overloadNotificationSend / config.overloadEarlyWarningSeconds() checks should just be in here instead then.